log.go

```diff
@@ -2,11 +2,45 @@ package wrapplog
 
 import (
 	"os"
+	"path"
 
 	"github.com/Sirupsen/logrus"
 )
 
+type WrappHook struct {
+	ServiceName string
+}
+
+func (w *WrappHook) Levels() []logrus.Level {
```

Comment:
It is required by logrus.Hook interface. The only reason we need this is to follow this interface.